ansible_deployer/command_line.py

```diff
@@ -242,13 +246,24 @@ def validate_option_values_against_config(config: dict, options: dict, required_
                         index = config["infra"].index(item)
                 selected_items["stage"] = validate_option_by_dict_with_name(options["stage"],\
                                                                 config["infra"][index]["stages"])
+            elif option == "limit":
+                for item in config["tasks"]["tasks"]:
+                    if item["name"] == options["task"]:
+                        if item["allow_limit"]:
+                            selected_items["limit"] = options["limit"]
+                            break
+                else:
+                    logger.error('Limit "%s" is not available for task "%s".', options["limit"],
+                                options["task"])
```

Comment:
it looks like missing space, we also use double quotes " for logger in other places I'd say we should follow that as current standard